postgres/datadog_checks/postgres/metadata.py

```diff
@@ -141,7 +141,7 @@
 
 PARTITION_ACTIVITY_QUERY = """
 SELECT pi.inhparent :: regclass         AS parent_table_name,
-       SUM(psu.seq_scan + psu.idx_scan) AS total_activity
+       COALESCE(SUM(psu.seq_scan + psu.idx_scan), 0) AS total_activity
```

Comment:
This will return `0` when there are sequential scans but no index scans and vice versa. The following expression should return a value even if one of the summands is `null`.
```suggestion
       SUM(COALESCE(psu.seq_scan, 0)) + SUM(COALESCE(psu.idx_scan, 0)) AS total_activity
```